src/monorepo-shipit/README.md

```diff
@@ -1,4 +1,4 @@
-Monorepo Shipit takes care of exporting and importing our source-codes from our private GitLab monorepo into any other Git repository. It can export even from our monorepo to another monorepo. We use it open-source some of our packages to our [GitHub](https://github.com/kiwicom). This way we can develop just like we are used to in one monorepo but we can contribute back to the community by making some of our codes open.
+Monorepo Shipit takes care of exporting and importing our source-codes from our private GitLab monorepo into any other Git repository. It can export even from our monorepo to another monorepo. We use it open-source some of our packages to our [GitHub](https://github.com/adeira). This way we can develop just like we are used to in one monorepo but we can contribute back to the community by making some of our codes open.
```

Comment:
> exporting and importing our source-codes from our private GitLab monorepo

This is no longer on gitlab is it?

Reply:
I didn't focus on the text itself - it will take some time to settle in. Feel free to create an issue or better send a PR. :)